spec/system/user_uses_the_editor_spec.rb

```diff
@@ -98,11 +98,12 @@ def fill_markdown_with(content)
   describe "using v2 editor", js: true do
     it "fill out form with rich content and click publish" do
       visit "/new"
-      fill_in "article-form-title", with: "This is a <span> test"
-      fill_in "tag-input", with: "What, Yo"
-      fill_in "article_body_markdown", with: "Hello"
+      within "form#article-form" do
+        fill_in "article-form-title", with: "This is a <span> test"
+        fill_in "tag-input", with: "What, Yo"
+        fill_in "article_body_markdown", with: "Hello"
+      end
       find("button", text: /\APublish\z/).click
```

Comment:
I left the click button out (it's probably correct to move it inside the block, and sufficient to only have the title check inside the `within` block since it triggers the waiting).